x-pack/legacy/plugins/security/server/lib/authentication/providers/oidc.ts

```diff
@@ -8,9 +8,9 @@ import Boom from 'boom';
 import type from 'type-detect';
```

Comment:
__note:__ OIDC tests are disabled :/ And we should enable them as soon as possible https://github.com/elastic/kibana/issues/36959. But for now I tweaked kbn-es locally and run the tests - they passed with the latest version of this PR.